### DISCUSSION TITLE
support create new doc from templates

### DISCUSSION THREAD

**Discussion by jinhefeng** [toeverything/AFFiNE#6478] [Ideas]
### Description

1. templates manager include create, edit, del and use.
2. template remain properties info.

### Use case

such as:
1. Meeting Minutes Template
2. Interview Record Template
3. Project Management Template

### Anything else?

like Microsoft Word

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

[a comment by an automated account is not shown]

**Comment by pengx17** [comment]
I think this is already on our roadmap. cc @hwangdev97

**Reply by aiyou9** [reply]
When will the calendar's preset template function be launched? How to deal with existing diaries?

**Comment by hwangdev97** [comment]
We will support document-level templates.However, the specific timing has yet to be scheduled.🤔
